src/admin.tsx

```diff
@@ -171,24 +285,46 @@ export default function AdminPage() {
           aria-label="file system navigator"
           defaultCollapseIcon={<ExpandMoreIcon />}
           defaultExpandIcon={<ChevronRightIcon />}
-          sx={{ height: '100%', flexGrow: 1, maxWidth: 600, overflowY: 'auto', paddingLeft: 10 }}
+          sx={{ height: '100%', flexGrow: 1, maxWidth: 900, overflowY: 'auto', paddingLeft: 10 }}
         >
           {elements}
+          <TreeItem
+            sx={{ padding: 1 }}
+            nodeId='add_category'
+            expandIcon={<TreeIcon tooltipText='New Category' icon={<AddIcon />} />}
+            collapseIcon={<TreeIcon tooltipText='Close Editor' icon={<BuildCircleIcon color='error' />} />}
+          >
+            <Typography variant='h6'><i>New Ambiance Category</i></Typography>
+            <Divider />
+            <CategoryEditor addCategory={addCategory}></CategoryEditor>
+          </TreeItem>
         </TreeView>
       </div>
     )
   }
 
-
   return (
     <div id="admin-page" data-testid='admin'>
-      <Box sx={{ flexGrow: 1 }}>
-        <AppBar className="app-bar" >
+      <Snackbar
+        open={open}
+        autoHideDuration={6000}
```

Comment:
Lets pull out this constant to a variable at the top of this file

Reply:
Can we use the constants.ts file? That's where I put it for now, but let me know if you disagree!